src/app/components/driver/driver.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
+import { MarkInactiveDriverService } from '../../services/mark-inactive-driver-service/mark-inactive-driver.service';
+import { Router } from '@angular/router';
+import { User } from 'src/app/models/user';
+import { Batch } from 'src/app/models/batch';
+
+
+
+@Component({
+  selector: 'app-driver',
+  templateUrl: './driver.component.html',
+  styleUrls: ['./driver.component.css']
+})
+export class DriverComponent implements OnInit {
+
+  userdriver : User ;
+
+  riders: User[];
+
+  userId = ''; 
```

Comment:
Why use all of these instance variables rather than just using the instance variables of the user that you already have? Assign userdriver to an instance of user with these values. Also, keep in mind naming conventions for things like "userDriver" and "markInactiveDriverService"